contracts/Super1155.sol

```diff
@@ -259,13 +254,18 @@ contract Super1155 is PermitControl, ERC165Storage, IERC1155, IERC1155MetadataUR
     @param _uri The metadata URI to perform later token ID substitution with.
     @param _proxyRegistryAddress The address of a proxy registry contract.
   */
-  constructor(string memory _name, string memory _uri,
+  constructor(address _owner, string memory _name, string memory _uri,
     address _proxyRegistryAddress) {
 
     // Register the ERC-165 interfaces.
     _registerInterface(INTERFACE_ERC1155);
     _registerInterface(INTERFACE_ERC1155_METADATA_URI);
 
+    setPermit(msg.sender, UNIVERSAL, CONFIGURE_GROUP, MAX_INT);
```

Comment:
We don't use [meta transactions](https://eips.ethereum.org/EIPS/eip-2771) and don't have plans to yet, but since we've access to the OpenZeppelin `Context` we might as well maintain compatibility by using their `_msgSender()` abstraction instead of `msg.sender` directly.